src/elements/Button.tsx

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
-import { StyleSheet, Text, TouchableOpacity } from 'react-native';
+import {
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  ViewStyle,
+  TouchableOpacityProps
+} from 'react-native';
 import theme from '../utils/theme';
 
-export default ({ children, highlight, style, ...props }) => {
+export interface Props extends TouchableOpacityProps {
```

Comment:
If we're exporting this interface, we should probably give it a more descriptive name then right? Like `ButtonProps`? So that we do not pollute the global namespace. Btw, why do you export in this case? Are those typings required when using this component, or did you rather want to create a generic Prop type to be reused across components?

Reply:
Good point, these props are component specific. So exporting is not and renaming will not be necessary.